### PR TITLE
Extended to support Configurations added in Azure & with that test results can be categorized based on the config used dev/stage/prod

### PR DESCRIPTION
**Pull request by sundarlogan**
Tests Defined: 3 (demo-test-1, 2 & 3)
image

Configurations: 3 (dev/stage/prod) 
image

Tests Execute: section shows 3 tests with 3 different configurations assigned to suite [Total test points: 9]
image

Test case results History: General all config shown
image

Test point History: Anyone config dev/stage/prod shown
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "azuredevops-test-reporter-ts",
-  "version": "1.0.2",
+  "version": "1.1.0",
```
Comment (vivek-dhayalan) [inline package.json:3]: @sundarlogan we should start thinking of having semantic version

File: test/testCasePoints.test.ts [hunk 2]
```diff
@@ -13,6 +13,16 @@ const azureConfig: IAzureConfig = {
   runName: 'sample',
 }
 
+const azureConfigWithConfigName: IAzureConfig = {
+  pat: 'v3e3pg4njfmzzcuoxwb4vja3trwbyafim4x4oww7sbcfbopfb3bq',
```
Comment (vivek-dhayalan) [inline test/testCasePoints.test.ts]: Can we have the token as part of env variable?

File: package.json (hunk 1 above)
Comment (sundarlogan) [reply package.json:3]: yes, added it.

File: test/testCasePoints.test.ts (hunk 2 above)
Comment (sundarlogan) [reply test/testCasePoints.test.ts]: yes, moved it to env.

**Review by vivek-dhayalan — APPROVED** [review]

**Pull request merged**